### ISSUE TITLE
Is the dataset shuffling enough?  

### ISSUE THREAD

**Issue by YoungYeeee** [ruohoruotsi/LSTM-Music-Genre-Classification#15]
When i get all the data together, and reshuffle the dataset. I get the better accuracy in LSTM.
this is the code:
```python
from sklearn.model_selection import train_test_split
X = torch.concatenate((train_X,test_X,dev_X),axis=0)
y = torch.concatenate((train_Y,test_Y,dev_Y),axis=0)
train_X, test_X, train_Y, test_Y = train_test_split(X, y, test_size=0.2, random_state=42)
```
